DEVELOPER_GUIDE.md

```diff
@@ -74,6 +83,74 @@ This is typically created in PyCharm IDE by visiting the `Python Interpreter`, s
 `
 In order to run tests within the PyCharm IDE, ensure the `Python Integrated Tools` / `Testing` / `Default Test Runner` is set to `pytest`.
 
+## Running Workloads
+
+### Installation  
```

Comment:
I'd rephrase this to `OpenSearch Installation`, just cause this developer guide pertains to OSB and we already have an "Installation" section